docs/Developer Reference/user-guides/acquire-link.md

```diff
@@ -10,19 +10,21 @@ This page will show you how to obtain and add testnet LINK to MetaMask. If you a
 
 # Testnet LINK Faucet
 
-* Navigate to the Chainlink Kovan Faucet site: <a href="https://kovan.chain.link/" target="_blank" rel="noreferrer, noopener">https://kovan.chain.link</a>
+* Navigate to the Chainlink Kovan Faucet site: <a href="https://faucets.chain.link/kovan" target="_blank" rel="noreferrer, noopener">https://faucets.chain.link/kovan</a>
 
 > 👍 Note
 > 
-> You can also get testnet LINK from: [Rinkeby](https://rinkeby.chain.link/)
+> You can also get testnet LINK from: [Rinkeby](https://faucets.chain.link/rinkeby)
```

Comment:
Kind of feel like this shouldn't be here, or we should give the full list of networks you can use